module-2/visualizing-real-world-data/your-code/Visualizing Real world data.ipynb

```diff
@@ -423,32 +423,40 @@
   },
```

Comment:
- Age is not exactly a numeric variables for relationships analysis purposes. This is the reason why you get such scatter plot. We can consider here as a categorical variable and you can explore the distribution of median income by age or age range (transforming the raw data) using a barplot, for example. 
- Scatter plots are used to show the relationship pattern between the dependent variable and the independent variable. It's a good practice to plot before the correlation matriz because it can give us an idea of which variables can be included in a scatter. 
- For this 
```Python
a = pd.plotting.scatter_matrix(plott)
```
You can also use a seaborn pairplot. Depending on your data and your audience.